iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/repository/HouseTableRepositoryImpl.java

```diff
@@ -107,23 +109,20 @@ public HouseTable save(HouseTable entity) {
     CreateUpdateEntityRequestBodyUserTable requestBody =
         new CreateUpdateEntityRequestBodyUserTable().entity(houseTableMapper.toUserTable(entity));
 
-    return getHtsRetryTemplate(Arrays.asList(IllegalStateException.class))
-        .execute(
-            context ->
-                apiInstance
-                    .putUserTable(requestBody)
-                    .map(EntityResponseBodyUserTable::getEntity)
-                    .map(houseTableMapper::toHouseTable)
-                    .onErrorResume(this::handleHtsHttpError)
-                    .block(Duration.ofSeconds(REQUEST_TIMEOUT_SECONDS)));
+    return apiInstance
```

Comment:
any mock test that can be added for this?

Reply:
I modified a test to guarantee that no retries are done for table saves